src/transactionsReplayLoop/index.ts

```diff
@@ -1,17 +1,18 @@
 import { loadEnv, unload } from '../secrets';
-import { tdhLoop } from '../tdhLoop/index';
+import { Transaction } from '../entities/ITransaction';
+import { transactions } from '../transactionsLoop/index';
+// import { LabTransaction } from '../entities/ITransaction';
+// import { transactions } from '../meme_lab';
 import { Logger } from '../logging';
 
 const logger = Logger.get('TRANSACTIONS_REPLAY_LOOP');
 
 export const handler = async (event?: any, context?: any) => {
-  const fromBlock = 16485000;
+  const fromBlock = 0;
   const toBlock = undefined;
   logger.info(`[RUNNING] [FROM BLOCK ${fromBlock}] [TO BLOCK ${toBlock}]`);
-  await loadEnv();
-  // await transactions(fromBlock, toBlock);
-  // await findOwnerMetrics();
-  await tdhLoop(true);
+  await loadEnv([Transaction]);
+  await transactions(fromBlock, toBlock);
```

Comment:
Would be nicer if this functions name would be verb instead of a noun. A verb which would describe what it does. I as unfamiliar reader of this code don't know what it does. So something like `fetchAndPersistTransactions` maybe?